WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsFormViewModel.swift

```diff
@@ -140,14 +140,13 @@ private extension WooShippingCustomsFormViewModel {
         .map { input -> ITNValidationError? in
             let (itn, items, hsTariffNumberTotalValueDictionary, countryCode) = input
             guard itn.isEmpty else {
-                return itn.isValidITN ? nil : .invalidFormat
+                return ITNNumberValidator.isValid(itn) ? nil : .invalidFormat
             }
 
             let totalItemValue = items.reduce(0, { sum, item in
                 sum + item.totalValue
             })
-            if hsTariffNumberTotalValueDictionary.isEmpty,
```

Comment:
This dictionary was used to check classes above $2500. We should revise the logic [here](https://github.com/woocommerce/woocommerce-shipping/blob/53ff1755d4cff2c3a5f6a414be4ce1c72f69d0f0/client/components/label-purchase/customs/validators.ts#L199) to see if this rule still applies, otherwise we should remove the dictionary if unused.